pkg/utils/kubelessutil.go

```diff
@@ -319,6 +320,7 @@ func populatePodSpec(funcObj *kubelessApi.Function, lr *langruntime.Langruntimes
 	if len(imagePullSecrets) > 0 {
 		result.ImagePullSecrets = imagePullSecrets
 	}
+	//if funcObj.Spec.InitContainers[0] != "" && funcObj.Spec.InitContainers[0].Resources
```

Comment:
You forgot to remove this line?

Reply:
removed the comment and updated the docs